src/Room/Queue.js

```diff
@@ -17,11 +21,18 @@ const Queue = ({queue}) => {
                             <Card.Title className="small">{queueTrack.track.title}</Card.Title>
                             <Card.Subtitle
                                 className="mb-2 small text-muted">{queueTrack.track.artist}</Card.Subtitle>
-                            <Card.Text className="small">Added by {queueTrack.user.name}</Card.Text>
+                            <Card.Text className="small">Added by {queueTrack.user.name} </Card.Text>
+                        </Card.Header>
+                        <Card.Header style={{backgroundColor: "white", borderWidth: 0}}>
+                            <Button variant="danger"
+                                    disabled={user.role !== "host" && queueTrack.user.id !== user.id}
```

Comment:
should this be `disabled={user.role !== "host" || queueTrack.user.id !== user.id}` (or instead of and)?  meaning, you can't delete from queue is either a) you're not a host, or b) (you're a guest and) you did not add the track.

Reply:
I think it is AND because OR would be satisfied for either condition. Example for host:

disabled ? --> False || True evaluates to True, meaning it would be disabled, even though I am the host

disabled ? --> False && True evaluates to False, meaning I would be able to delete as a host